### PR TITLE
Release 2.6.0: add members.get and testing

### PR DESCRIPTION
**Pull request by mtompset**
### Add
* members.get added with a test

### CODE REVIEW

File: SilMock/Google/Service/Directory/Resource/Members.php [hunk 1]
```diff
@@ -15,6 +15,19 @@ public function __construct(?string $dbFile = null)
...
+    {
+        $members = $this->listMembers($groupKey);
+        $memberList = $members->getMembers();
+        foreach ($memberList as $member) {
+            $memberEmailAddress = mb_strtolower($member->getEmail());
+            if ($memberEmailAddress === mb_strtolower($memberKey)) {
+                return $member;
+            }
+        }
+        throw new Exception('Member not found');
```
Comment (forevermatt) [inline SilMock/Google/Service/Directory/Resource/Members.php:28]: The docs aren't clear about what the actual Google API does in this case. Do you know if it also throws some kind of exception if the member is not found?

File: SilMock/tests/Google/Service/Directory/Resource/MembersTest.php [hunk 2]
```diff
@@ -36,6 +36,24 @@ public function testInsert()
         self::assertTrue($addedMember instanceof GoogleDirectory_Member);
     }
 
+    public function testGet()
+    {
+        $groupEmailAddress = 'sample_group@groups.example.com';
+        $emailAddress = 'test@example.com';
+        $mockGoogleServiceDirectory = new GoogleMock_Directory('anyclient', $this->dataFile);
```
Comment (forevermatt) [inline SilMock/tests/Google/Service/Directory/Resource/MembersTest.php:43]: It seems like this test assumes (but doesn't cause/verify) that the given email address is a member of the given group. Do you think it's worth adding code (now or later) to ensure that is the case as part of the "setup" part of this test (i.e. before the call to "members->get()")?

**Review by forevermatt — APPROVED** [review]

File: SilMock/Google/Service/Directory/Resource/Members.php (hunk 1 above)
Comment (mtompset) [reply SilMock/Google/Service/Directory/Resource/Members.php:28]: True, but given your more current "throw exception unless it works" philosophy seems more modern and in line with Google's current thinking.

File: SilMock/tests/Google/Service/Directory/Resource/MembersTest.php (hunk 2 above)
Comment (mtompset) [reply SilMock/tests/Google/Service/Directory/Resource/MembersTest.php:43]: This is why the test order is important currently, which is not perfection, but right now it needs to work.

**Pull request merged**